### PR TITLE
Fix an IndexOutOfRangeException in EnumCompletion

### PR DESCRIPTION
**Pull request by rchande**
Fixes internal 201807.

### CODE REVIEW

File: src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/EnumCompletionProviderTests.vb [hunk 1]
```diff
@@ -460,6 +460,14 @@ End Class
             Await VerifyProviderCommitAsync(markup, "E.A", expected, ","c, textTypedSoFar:="")
         End Function
 
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(201807, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=201807&triage=true&_a=edit")>
+        Public Async Function TestDoNotCrashAtPosition1AfterEquals() As Task
+            Dim markup = <Text><![CDATA[=$$     
+]]></Text>.Value
```
Comment (rchande) [inline src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/EnumCompletionProviderTests.vb]: There are additional spaces after the $$.

**Comment by rchande** [comment]
@dotnet/roslyn-ide review please.

File: src/Features/VisualBasic/Portable/Completion/CompletionProviders/EnumCompletionProvider.vb [hunk 2]
```diff
@@ -64,7 +64,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Friend Overrides Function IsInsertionTrigger(text As SourceText, characterPosition As Integer, options As OptionSet) As Boolean
             Return text(characterPosition) = " "c OrElse
                 text(characterPosition) = "("c OrElse
-                (text.Length > 2 AndAlso text(characterPosition) = "="c AndAlso text(characterPosition - 1) = ":"c) OrElse
+                (characterPosition > 1 AndAlso text(characterPosition) = "="c AndAlso text(characterPosition - 1) = ":"c) OrElse
```
Comment (CyrusNajmabadi) [inline src/Features/VisualBasic/Portable/Completion/CompletionProviders/EnumCompletionProvider.vb]: how do you know text is long enough in this case?

Comment (rchande) [inline src/Features/VisualBasic/Portable/Completion/CompletionProviders/EnumCompletionProvider.vb]: The position is within the bounds of the text, and we're only looking backward from it.

Comment (CyrusNajmabadi) [inline src/Features/VisualBasic/Portable/Completion/CompletionProviders/EnumCompletionProvider.vb]: ah ok.

**Comment by CyrusNajmabadi** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**